django_ckeditor_5/widgets.py

```diff
@@ -81,6 +83,11 @@ def render(self, name, value, attrs=None, renderer=None):
         context["config"] = self.config
         context["script_id"] = "{}{}".format(attrs["id"], "_script")
         context["upload_url"] = reverse("ck_editor_5_upload_file")
+        context["upload_file_types"] = json.dumps(getattr(
+            settings,
+            "CKEDITOR_5_UPLOAD_FILE_TYPES",
+            ["jpeg", "png", "gif", "bmp", "webp", "tiff"],
```

Comment:
Maybe we can make this array default, but add the option to extend it in the settings?